webapp/src/modules/bid/selectors.js

```diff
@@ -46,6 +46,10 @@ export const getBidByAssetIdFactory = (assetId, assetType) =>
   )
 
 export const getWalletBidsByAsset = (state, asset, assetType) => {
+  if (!asset) {
+    return []
+  }
```

Comment:
Why is this needed?

Reply:
Because we are using the selector when you create an Estate and the asset is undefined. I should add a comment. What do you think?